wordpress-mobile/gutenberg-mobile
# Refactor the spacer block.

Refactor the spacer block on mobile to have the same edit code as the web. All the work was done on the Gutenberg PR bellow:

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/20746

To test:
 - Start the demo app
 - Add a spacer block
 - Check if the block settings are working correctly and updating the block view.

PR submission checklist:
